arviz/plots/backends/bokeh/distplot.py

```diff
@@ -119,8 +128,87 @@ def _histplot_bokeh_op(values, values2, rotated, ax, hist_kwargs):
     if hist_kwargs.pop("cumulative", False):
         hist = np.cumsum(hist)
         hist /= hist[-1]
-    if rotated:
-        ax.quad(top=edges[:-1], bottom=edges[1:], left=0, right=hist, **hist_kwargs)
+
+    if is_circular:
+
+        if is_circular == "degrees":
+            edges = np.deg2rad(edges)
+
+        start_angle = edges
```

Comment:
instead of renaming variables here you may want to 

outer_radius, start_angle = np.histogram(np.asarray(values).flatten(), density=density, bins=bins)

just use the name "edges" and "hist" (I think this may help to make the code easier to read, as probably most people is more used to what "edges" and "hist" means. But I do not have a strong opinion here.